lib/liquid_voting/delegations.ex

```diff
@@ -115,10 +115,22 @@ defmodule LiquidVoting.Delegations do
     |> Repo.insert()
   end
 
+  @doc """
+  Creates a delegation.
+
+  ## Examples
+
+      iex> create_delegation(%{field: value})
+      %Delegation{}
+
+      iex> create_delegation(%{field: bad_value})
+      Ecto.*Error
+
+  """
   def create_delegation!(attrs \\ %{}) do
     %Delegation{}
     |> Delegation.changeset(attrs)
-    |> Repo.insert()
+    |> Repo.insert!()
```

Comment:
For the future - this is a needed change, but would have been better as a separate PR, because it's changing behavior and not just docs.